plugins/modules/dconf.py

```diff
@@ -21,11 +21,22 @@
   - Since C(dconf) requires a running D-Bus session to change values, the module
     will try to detect an existing session and reuse it, or run the tool via
     C(dbus-run-session).
+requirements:
+  - C(gi.repository) Python library (usually included in the OS on hosts which
+    have C(dconf))
```

Comment:
```suggestion
  - Optionally the C(gi.repository) Python library (usually included in the OS on hosts which
    have C(dconf)); this will become a non-optional requirement in a future major release of community.general
```